src/foam/nanos/crunch/CapabilityAvailabilityDAO.js

```diff
@@ -61,6 +70,12 @@ foam.CLASS({
     {
       name: 'select_',
       javaCode: `
+        Theme theme = (Theme) x.get("theme");
+        if ( theme != null && theme.getRestrictedCapabilities() != null ) {
+          return getDelegate()
+            .where(foam.mlang.MLang.NOT(foam.mlang.MLang.IN(Capability.ID, theme.getRestrictedCapabilities())))
```

Comment:
BTW, MLang's are predicates, so you could have used this NOT(IN()) mlang in the find to have avoided copying the array.